Packs/Jizo_M/Integrations/JizoM/JizoM.yml

```diff
@@ -0,0 +1,282 @@
+category: Network Security
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: JizoM
+  version: -1
+name: JizoM
+display: JizoM
+description: This integration ensures interaction with the JizoM API.
+configuration:
+- section: Collect
+  display: Username
+  displaypassword: Password
+  name: credentials
+  type: 9
+  required: true
+  additionalinfo: Reliability of the source providing the intelligence data.
+- section: Connect
+  display: Server URL. e.g., https://127.0.0.1:9001
+  name: url
+  type: 0
+  required: true
+- section: Collect
+  display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- section: Connect
+  display: Incident type
+  name: incidentType
+  defaultvalue: Jizo Alert
+  type: 13
+  required: false
+- section: Connect
+  advanced: true
+  display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- section: Connect
+  advanced: true
+  display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  type: 0
+  required: false
+- display: First fetch time (<number> <time unit>, for example, 12 hours, 7 days, 3 months, 1 year)  
+  name: first_fetch
+  defaultvalue: 3 days
+  type: 0
+  required: false
+- display: Maximum number of alerts per fetch
+  name: max_fetch
+  defaultvalue: "10"
+  type: 0
+  required: false
+script:
+  type: python
+  commands:
+  - name: jizo-m-protocols-get
+    arguments:
+    - name: ip_src
+      description: Ipv4 or Ipv6 of the source.
+      type: textArea
+    - name: ip_dest
+      description: Ipv4 or Ipv6 of the destination.
+      type: textArea
+    - name: datetime_from
+      description: Get the alerts that were occurred from this date, for example, "3 days ago", "2020-01-01-00:00:00". The default value is 7 days ago.  
+      type: textArea
+    - name: datetime_to
+      description: Get the alerts that were occurred up to this date, for example, "3 days ago", "2020-01-01-00:00:00". The default is now.
+      type: textArea
+    - name: probe_name
+      description: The name of the jizo probe.
+      type: textArea
+    - name: page
+      description: A page number for pagination.
+      type: keyValue
+    - name: limit
+      description: The maximum number of protocols to display per alert.
+      type: keyValue
```

Comment:
Is there a specific reason that the arguments page & limit are of type `KeyValue`?
Please update them to be `textArea` in all places.